components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/util/APIClient.java

```diff
@@ -50,6 +53,9 @@ public class APIClient {
     private static final Log LOG = LogFactory.getLog(APIClient.class);
     private static final ActionExecutionDiagnosticLogger DIAGNOSTIC_LOGGER = new ActionExecutionDiagnosticLogger();
     private final CloseableHttpClient httpClient;
+    private static final String ACTION_STATUS = "actionStatus";
+    private static final String SERVER_ERROR_CODE = "server_error";
+    private static final String SEVER_ERROR_MESSAGE = "Internal Server Error";
```

Comment:
These constants are not used. Isn't it?

Reply:
This has been removed with e01617689c045e164749dedaaf2f7f0603920ae2